debian/control

```diff
@@ -3,6 +3,11 @@ Section: admin
 Priority: extra
 Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
 Build-Depends: debhelper (>= 9),
+               libxml2-dev,
+               libxslt-dev,
+               python-dev,
+               lib32z1-dev,
```

Comment:
Where is travis failing to building lxml? This line shows it installing and building against our master branch

https://travis-ci.org/repositories#L147